.github/workflows/create-release.yaml

```diff
@@ -1,24 +1,45 @@
 ---
-name: Create Release
+name: Create release
+
 on:
   workflow_dispatch:
     inputs:
-      input_var:
-        description: "Say Hi!"
-        required: true
+      release_tag:
```

Comment:
Since the script is enough smart to create the tag by itself i would suggest to remove the release_tag, since we will never use it.